docs/getting-started.md

```diff
@@ -18,9 +18,19 @@ npm start
 
 Then open [http://localhost:3000/](http://localhost:3000/) to see your app.
 
+**Option 2:** If you would like to scaffold from 1 example, please see [Running an Example](./building-your-application/available-scripts/running-example.md)
+
+**Option 3:** You can also clone the boilerplate repo to get all the examples. Then run `npm i` followed by `npm start` from the example directory you want to run.
```

Comment:
Maybe can can link https://github.com/NMFS-RADFish/boilerplate to `boilerplate repo` text so it's easier for the reader to find